examples/s3/late-init-bucket.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: s3.aws.crossplane.io/v1beta1
```

Comment:
Do we need two examples? Which one do you think shows off the most functionality?

Reply:
The main difference between the two is the inclusion of the external-name annotation. The old bucket probably shows more functionality, and I can remove the late-init file if you think that's a good idea